html/settings/_settings.scss

```diff
@@ -1532,6 +1532,41 @@ $sprk-toggle-trigger-font-weight-small: normal !default;
 $sprk-toggle-transition-timing: 0.3s ease !default;
 $sprk-toggle-transistion-timing: $sprk-toggle-transition-timing; //TODO: deprecate
 
+//
+// Tooltip
+///
+
+/// The border for the tooltip trigger.
+$sprk-tooltip-trigger-border: none !default;
+/// The background color for the tooltip trigger.
+$sprk-tooltip-trigger-background-color: transparent !default;
+/// The padding for the tooltip trigger.
+$sprk-tooltip-trigger-padding: 0 !default;
+/// The text align for the tooltip trigger.
+$sprk-tooltip-trigger-text-align: left !default;
+/// The background color for the tooltip.
+$sprk-tooltip-background-color: $sprk-black !default;
+/// The font color for the tooltip.
+$sprk-tooltip-font-color: $sprk-white !default;
+/// The padding for the tooltip.
+$sprk-tooltip-padding: 12px $sprk-space-m !default;
```

Comment:
Hey @bobwise , can we use one of these settings that we already have? Maybe $sprk-space-inset-short-m? I would bring it up to jane to ask

/// Value for the extra small inset short space.
$sprk-space-inset-short-xs: 2px 4px 2px 4px !default;
/// Value for the small inset short space.
$sprk-space-inset-short-s: 4px 8px 4px 8px !default;
/// Value for the medium inset short space.
$sprk-space-inset-short-m: 8px 16px 8px 16px !default;
/// Value for the large inset short space.
$sprk-space-inset-short-l: 16px 32px 16px 32px !default;
/// Value for the extra large inset short space.
$sprk-space-inset-short-xl: 32px 64px 32px 64px !default;